pkg/cmd/open/open.go

```diff
@@ -23,6 +23,7 @@ var nameURLmap = map[string]string{
 	"docs":      "https://algolia.com/doc/",
 	"cli-docs":  "https://algolia.com/doc/tools/cli/get-started/overview/",
 	"cli-repo":  "https://github.com/algolia/cli",
+	"status":    "https://status.algolia.com/",
```

Comment:
I would say we could also open the dashboard on `https://www.algolia.com/apps/<appID>/monitoring/status` page.
Basically, if a profile / appID is currently configured / set as default, open the dashboard status page.
If no profile / no appID, open `https://status.algolia.com/`. WDYT?

Reply:
The status page on the dashboard is more informative IMHO, as it clearly state which cluster you are and the status of the cluster.

Reply:
I agree, It'd be even better when we'll handle authentication with the CLI, we could force login to the browser by passing a token (maybe?) so we're sure the user will never have to enter his login/password to see the `https://www.algolia.com/apps/<appID>/monitoring/status` page